index.html

```diff
@@ -0,0 +1,117 @@
+                        </a>
+                        <a href="#">
+                            <li class="ms-5 me-3">News</li>
+                        </a>
+                        <a style="border: 3px solid rgb(255, 51, 0); border-radius: 5px;color: rgb(255, 51, 0);"
+                            href="#">
+                            <li class="mx-5">Campaign</li>
+                        </a>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+
+        <div class="container text-dark my-2 pb-5 sm-fs-4">
+            <h3 class="pt-5 pb-2 mt-1" style="color: rgb(255, 51, 0); font-weight: 400; font-size: 32px;">"Hello, Sharing world"</h3>
+            <h1 style="color: rgb(255, 51, 0);">CONTAINER FLUID</h1>
+            <h1 style="color: rgb(255, 51, 0);">ARTS FESTIVAL</h1>
+            <h1 style="color: rgb(255, 51, 0);">2022</h1>
+            <p class="bg-light my-5 p-1">Container Festival is without doubt the Bucaramanga's most famous music and
+                performing arts festival, if not the
+                world's. Embracing every genre across the spectrum </p>
+            <h1 class="date">2022.11.03(THU) ~ 06(SAT)</h1>
+            <small class="pb-5">@National Museum, Modern Art Museum and more</small>
+        </div>
+    </div>
+
+    <div class="programwrapper">
+        <h1 class="text-center fs-3 " style="color: #fff;">Main Program</h1>
+        <hr class='mb-5'
+            style="color:rgb(255, 51, 0,1) ;border: 2px solid rgb(255, 51, 0,1);width: 25%; margin-left: 37%;">
+        <div class="mainprogram">
+        </div>
+        <div class="wholeprogrambutton"></div>
+    </div>
+
+    <div class="speakerswrapper">
+        <h1 class="text-center fs-3 mt-5" style="color: rgb(0, 0, 0);">Speakers</h1>
+        <hr class="mb-5" style="color:rgb(255, 51, 0) ;border: 2px solid rgb(255, 51, 0);width: 25%; margin-left: 37%;">
+        <div class="speakers">
+        </div>
+    </div>
+
+    <div class="open">
+        <div class="close">
+            <a href="#"> <img src="img/Icon.png" alt="x icon"></a>
+        </div>
+    </div>
+
+    <div class="sponsordiv">
+        <h1 class="text-center fs-3 mt-5" style="color: rgb(255, 255, 255);">Partners</h1>
+        <hr style="color:rgb(255, 51, 0) ;border: 2px solid rgb(255, 51, 0);width: 25%; margin-left: 37%;">
+
+        <div class="d-flex justify-content-center sponsor">
+            <img src="img/google-play.svg" alt="logo google play">
+            <img src="img/google.svg" alt="logo google">
+            <img src="img/youtube.svg" alt="logo youtube">
+            <img src="img/browser-chrome.svg" alt="logo chrome">
+            <img src="img/android.svg" alt="logo android">
+            <img src="img/android2.svg" alt="logo android 2">
+        </div>
+
+    </div>
+
+    <footer class="d-flex justify-content-center">
+        <img class="m-5" src="img/cc-logo2.png" alt="logo creative commons">
+    </footer>
```

Comment:
- [ ] Kindly add text to the right of the footer and the logo to the left to match the footer in the design guidelines 💯 (This applies to the about page too)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198692667-622af094-55f7-4e44-9c10-af7842a947d0.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/198692718-9cbabbd3-383c-4d2a-bebc-ca9f30c1f4da.png)